packages/docusaurus-theme-classic/src/theme/Toggle/styles.module.css

```diff
@@ -62,9 +63,6 @@
 
 :global(.react-toggle--disabled) {
   cursor: not-allowed;
-  opacity: 0.5;
```

Comment:
Removed this so it doesnt look transparent on first paint.
Cursor not allowed is needed for a11y, because anyway the toggle can't be clicked until reactjs load